
# References used in the presentation

**Jia, Dini-Andreote, Salles. ISME COMMUN. 2, 96 (2022)** - Jia, X., Dini-Andreote, F. & Salles, J.F. Unravelling the interplay of ecological processes structuring the bacterial rare biosphere. ISME COMMUN. 2, 96 (2022). https://doi.org/10.1038/s43705-022-00177-6

**McLaren, Willis, Callahan.  Elife. 2019** - Michael R McLaren, Amy D Willis, Benjamin J Callahan (2019) Consistent and correctable bias in metagenomic sequencing experiments eLife 8:e46923 https://doi.org/10.7554/eLife.46923

**McLaren et al.  biorxiv. 2022** - Michael R. McLaren, Jacob T. Nearing, Amy D. Willis, Karen G. Lloyd, Benjamin J. Callahan bioRxiv 2022.08.19.504330; doi: https://doi.org/10.1101/2022.08.19.504330

**Dimitrov et al.  PeerJ. 2017** - Dimitrov, Mauricio R., et al. "Successive DNA extractions improve characterization of soil microbial communities." PeerJ 5 (2017): e2915.

**Willis & Martin.  Biostatistics. 2022** - Amy D Willis, Bryan D Martin, Estimating diversity in networked ecological communities, Biostatistics, Volume 23, Issue 1, January 2022, Pages 207–222, https://doi.org/10.1093/biostatistics/kxaa015

**Manor & Borenstein.  Genome Biology. 2015** - Manor, O., Borenstein, E. MUSiCC: a marker genes based framework for metagenomic normalization and accurate profiling of gene abundances in the microbiome. Genome Biol 16, 53 (2015). https://doi.org/10.1186/s13059-015-0610-8

**Mardis.  Annu. Rev. Genom. Human Genet. 2008** - Mardis ER. Next-generation DNA sequencing methods. Annu Rev Genomics Hum Genet. 2008;9:387-402. doi: 10.1146/annurev.genom.9.081307.164359. PMID: 18576944.

**Gloor & Reid.  Can. J. Microbiol. 2016** - Gregory B. Gloor and Gregor Reid. Compositional analysis: a valid approach to analyze microbiome high-throughput sequencing data. Canadian Journal of Microbiology. 62(8): 692-703. https://doi.org/10.1139/cjm-2015-0821

**Leite & Kuramae.  SBB. 2020** - Leite, Márcio FA, and Eiko E. Kuramae. "You must choose, but choose wisely: Model-based approaches for microbial community analysis." Soil Biology and Biochemistry 151 (2020): 108042. https://doi.org/10.1016/j.soilbio.2020.108042

**Warton et al.  Methods Ecol. Evol. 2012** - Warton, David I., Stephen T. Wright, and Yi Wang. "Distance‐based multivariate analyses confound location and dispersion effects." Methods in Ecology and Evolution 3.1 (2012): 89-101.

**Warton & Hui.  Methods Ecol. Evol. 2017** - Warton, David I., and Francis KC Hui. "The central role of mean‐variance relationships in the analysis of multivariate abundance data: A response to Roberts (2017)." Methods in Ecology and Evolution 8.11 (2017): 1408-1414. https://doi.org/10.1111/2041-210X.12843

**Warton et al.  TREE. 2015** - Warton DI, Blanchet FG, O'Hara RB, Ovaskainen O, Taskinen S, Walker SC, Hui FKC. So Many Variables: Joint Modeling in Community Ecology. Trends Ecol Evol. 2015 Dec;30(12):766-779. doi: 10.1016/j.tree.2015.09.007. Epub 2015 Oct 28. PMID: 26519235.

**Ovaskainen et al.  Ecol. 2017** - Ovaskainen, Otso, et al. "How to make more out of community data? A conceptual framework and its implementation as models and software." Ecology letters 20.5 (2017): 561-576. https://doi.org/10.1111/ele.12757

**Zhou et al.  Environ. microbiome. 2021** - Zhou, X., Leite, M.F.A., Zhang, Z. et al. Facilitation in the soil microbiome does not necessarily lead to niche expansion. Environmental Microbiome 16, 4 (2021). https://doi.org/10.1186/s40793-021-00373-2

**Clark et al.  Ecol. Monogr. 2017** - Clark, James S., et al. "Generalized joint attribute modeling for biodiversity analysis: Median‐zero, multivariate, multifarious data." Ecological Monographs 87.1 (2017): 34-56. https://doi.org/10.1002/ecm.1241

**Bossolani et al.  SBB. 2021** - Bossolani, João W., et al. "Modulation of the soil microbiome by long-term Ca-based soil amendments boosts soil organic carbon and physicochemical quality in a tropical no-till crop rotation system." Soil Biology and Biochemistry 156 (2021): 108188. https://doi.org/10.1016/j.soilbio.2021.108188

# Links for the tutorials

**Generalized joint attribute modeling (GJAM)** - https://cran.r-project.org/web/packages/gjam/vignettes/gjamVignette.html#big-s-composition-data-fungal-endophytes
**DivNet** - https://github.com/adw96/DivNet
**MUSiCC** - https://github.com/borenstein-lab/MUSiCC
**Exploratory Compositional PCA Biplot** - https://github.com/ggloor/CoDa_microbiome_tutorial/wiki/Part-1:-Exploratory-Compositional-PCA-biplot
**Hmsc: high-dimensional multivariate models** - https://cran.r-project.org/web/packages/Hmsc/vignettes/vignette_3_multivariate_high.pdf
**BORAL** - https://besjournals.onlinelibrary.wiley.com/action/downloadSupplement?doi=10.1111%2F2041-210X.12514&file=mee312514-sup-0001-AppendixS1.pdf

